azure_translator/errors.py

```diff
@@ -0,0 +1,41 @@
+"""Azure errors module."""
+import xml.etree.ElementTree as ET
+
+
+class BaseAzureException(Exception):
+    """
+    Base exception for the SDK.
+    """
+    def __init__(self, *args, **kwargs):
+        self.response = kwargs.pop('response', None)
+        self.request = kwargs.pop('request', None)
+        super(BaseAzureException, self).__init__(*args, **kwargs)
+
+
+class AzureApiError(BaseAzureException):
```

Comment:
Need to clarify this bit − should that error parsing be used for all errors (base)?